lib/statsd_listener.rb

```diff
@@ -5,23 +5,24 @@
 class StatsdListener
   include Term::ANSIColor
 
-  def self.run(port)
+  def self.run(port: nil, host: nil)
     puts "Starting MiniStatsd...\n\n"
 
-    new(port).run
+    new(host: host, port: port).run
   end
 
-  def initialize(port:)
+  def initialize(host: nil, port: nil)
     $stdout.sync = true
 
     @socket = UDPSocket.new
     @port = PortSanitizer.sanitize(port)
+    @host = host
 
-    @socket.bind(nil, @port)
+    @socket.bind(@host, @port)
   end
 
   def run
-    puts "Listening on port #{@port}"
+    puts "Listening on #{@host}:#{@port}"
```

Comment:
Thank you for the suggestion @IgorMarques. I was trying to keep the gem with the same functionality. Previously the gem binds on `nil` and it seems that this doesn't default to 127.0.0.1. I have updated now to include a default, as when binding a UDP socket to nil, works in Ruby once. When trying a second time that fails, so this change should fix any multiple bindings too.